src/main/java/uk/gov/hmcts/darts/common/entity/DailyList.java

```diff
@@ -10,63 +10,96 @@
 import jakarta.persistence.SequenceGenerator;
 import jakarta.persistence.Table;
 import jakarta.persistence.Version;
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
 import java.time.OffsetDateTime;
 
 @Entity
-@Table(name = "moj_daily_list")
+@Table(name = DailyList.TABLE_NAME)
 @Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class DailyList {
 
+    public static final String ID = "moj_dal_id";
+    public static final String COURTHOUSE_ID = "moj_crt_id";
```

Comment:
Please be aware that Barry has changed this field to be `moj_cth_id` - see his latest v21.sql